src/btree/bt_read.c

```diff
@@ -68,6 +68,129 @@ __row_instantiate(WT_SESSION_IMPL *session,
 	return (0);
 }
 
+/*
+ * __wt_find_lookaside_upd --
```

Comment:
Why put `__wt_find_lookaside_upd()` in `bt_read.c`? Happy to believe it should be there, but I would have guessed `cache_las.c`?

Reply:
I based this function on ```__las_page_instantiate()``` which is in bt_read.c. I have moved ```__wt_find_lookaside_upd()``` into cache_las.c